src/libs/actions/BankAccounts.js

```diff
@@ -760,9 +785,10 @@ function setupWithdrawalAccount(data) {
             } else {
                 if (response.jsonCode === 666 || response.jsonCode === 404) {
                     error = response.message;
-                }
-
-                if (response.jsonCode === 402) {
+                } else if (JSON_CODE_TO_ERROR_KEYS[response.jsonCode]) {
+                    setBankAccountFormValidationErrors({[JSON_CODE_TO_ERROR_KEYS[response.jsonCode]]: true});
+                    setErrorModalVisible(true);
```

Comment:
This is an interesting approach, but I feel it maybe makes it difficult to tell which errors we are handling here.

I haven't thought of a great solution yet, but I think this is a pattern we're going to want to apply pretty regularly so we should think about it some more and maybe be cautious about optimizing too soon. 

I sort of imagined this being more explicit for now like...

```js
} else if (response.jsonCode === CONST.BANK_ACCOUNT.ERROR_CODE.INVALID_PHONE) {
    errors.phoneNumber = true;
} else if (response.jsonCode === CONST.BANK_ACCOUNT.ERROR_CODE.INVALID_DATE) {
    errors.date = true;
...

if (_.size(errors)) {
    showBankAccountFormValidationErrors(errors);
    setErrorModalVisible(true); 
}
```

This way we can look at the response handler and just really clearly see what kinds of things can happen.

Reply:
Updated. You may noticed that the `errors` variable is not defined in the closest scope, it is in a scope next to `error`. I made it like that because eventually it is supposed to be used with other steps. But if you think I should move it to the closest scope, let me know.